src/v2/Apps/Consign/Routes/NewMarketingLanding/Components/SoldRecentlyOnArtsy.tsx

```diff
@@ -0,0 +1,216 @@
+                  display="block"
+                  textDecoration="none"
+                >
+                  <ShelfArtworkFragmentContainer
+                    artwork={artwork!}
+                    key={artwork!.internalID}
+                    hidePartnerName
+                    hideSaleInfo
+                    lazyLoad
+                    // @ts-ignore
+                    contextModule={ContextModule.artworkRecentlySoldGrid}
+                    onClick={trackArtworkItemClick(artwork, index)}
+                  />
+
+                  <Flex
+                    mt={4}
+                    flexDirection="row"
+                    alignItems="center"
+                    justifyContent="space-between"
+                    color="black60"
+                  >
+                    <Text variant="xs">Estimate</Text>
+
+                    <Text variant="xs">
+                      {lowEstimate?.display}—{highEstimate?.display}
+                    </Text>
+                  </Flex>
+
+                  <Flex
+                    flexDirection="row"
+                    alignItems="center"
+                    justifyContent="space-between"
+                    color="blue100"
+                  >
+                    <Text variant="xs">Sold for (incl. premium)</Text>
+
+                    <Text variant="xs">{priceRealized?.display}</Text>
+                  </Flex>
+                </RouterLink>
+              </Fragment>
+            )
+          }
+        )
+      }}
+    />
+  )
+}
+
+const SoldRecentlyOnArtsyFragmentContainer = createFragmentContainer(
+  SoldRecentlyOnArtsy,
+  {
+    recentlySoldArtworks: graphql`
+      fragment SoldRecentlyOnArtsy_recentlySoldArtworks on RecentlySoldArtworkTypeConnection {
+        edges {
+          node {
+            artwork {
+              ...ShelfArtwork_artwork @arguments(width: 325)
+              slug
+              href
+              internalID
+            }
+            lowEstimate {
+              display
+            }
+            highEstimate {
+              display
+            }
+            priceRealized {
+              display
+            }
+          }
+        }
+      }
+    `,
+  }
+)
+
+const PLACEHOLDER = (
+  <Skeleton>
+    <Rail
+      title="Sold recently on Artsy"
+      getItems={() => {
+        return [...new Array(20)].map((_, i) => {
+          return (
+            <Box width={325} key={i}>
+              <SkeletonBox width={325} height={[200, 300, 250, 275][i % 4]} />
+              <SkeletonText variant="md">Artist Name</SkeletonText>
+              <SkeletonText variant="md">Artwork Title</SkeletonText>
+
+              <Spacer mt={4} />
+
+              <Flex
+                flexDirection="row"
+                alignItems="center"
+                justifyContent="space-between"
+              >
+                <SkeletonText variant="xs">Estimate</SkeletonText>
+
+                <SkeletonText variant="xs">USD 100,000 - 100,000</SkeletonText>
```

Comment:
**question:** Will users be able to see the text in this `SkeletonText`, or will they see a grey box that occupies the same amount as that text would?

Reply:
They will see a grey box
![Screenshot from 2022-05-02 15-32-42](https://user-images.githubusercontent.com/79979820/166234193-94efada4-9257-4ecb-9268-d45900083575.png)